alpha/external-web/package.json

```diff
@@ -9,11 +9,12 @@
   "scripts": {
     "start": "node start.js",
     "lint": "standard",
-    "test": "npm run lint"
+    "test": "npm run lint && node_modules/.bin/mocha -u exports test/**/*"
   },
   "standard": {
     "ignore": [
-      "jquery*.js"
+      "jquery*.js",
+      "test-*"
```

Comment:
Remove this. It is preventing standard from running on any test file.

We should replace this global ignore with something to ignore the testing libraries specifically.